src/renderer/store/battle.store.ts

```diff
@@ -74,10 +76,59 @@ function removeFromSpectators(participant: Player) {
     if (index !== -1) battleStore.spectators.splice(index, 1);
 }
 
+function removeTeam(teamId: number) {
```

Comment:
This part doesn't work properly still:

1. Start scav game
2. Add 3rd box
3. Add move scav to 3rd box
4. Add bot to 2nd box
5. Click on any of the "custom boxes" presets, that triggers removeTeam
6. Now team 2 has raptor and AI in the same team

It can be also done the other way around:
1. Add 3rd box
2. Fill 1st team with many bots so it always remains full
3. Move player to 3rd team
4. Remove 3rd box
5. Now player and scav are in the same team